charts/px-central/values.yaml

```diff
@@ -252,7 +258,7 @@ images:
     registry: pure-artifactory.dev.purestorage.com/px-docker-remote
     repo: portworx
     imageName: keycloak
-    tag: 26.2.4
+    tag: 26.2.4_v2
```

Comment:
Why did we change the keycloak version?

Reply:
Is it because we need all this?
```
COPY --from=nc-builder /bin/nc /usr/bin/nc
COPY --from=nc-builder /lib64/libssl.so.3 /lib64/
COPY --from=nc-builder /lib64/libcrypto.so.3 /lib64/
COPY --from=nc-builder /lib64/libpcap.so.1 /lib64/
COPY --from=nc-builder /lib64/libibverbs.so.1 /lib64/
COPY --from=nc-builder /lib64/libnl-route-3.so.200 /lib64/
COPY --from=nc-builder /lib64/libnl-3.so.200 /lib64/
```

Reply:
But then the makefile and dockerfile should be in the pxcentral-backend repo right? Is this the right place to keep it?

Reply:
The only difference is the nc binary , lets have it as a common for non mtls and mtls , will be easier to maintain and no need to pull extra image for aitgapped installation. 
Placed the dockerfile here so we wont miss it during any new releases , also keycloak is a third party image so having it in the pxcentral-backend will not bring any benefit from this place

Reply:
Have moved the dockerfile to the backend repo @mkoppal-px